docs/operations/druid-console.md

```diff
@@ -102,27 +117,30 @@ Click on the magnifying glass for any task to see more detail about it.
 
 ![tasks-status](../assets/web-console-09-task-status.png)
 
-## Servers
+## Services
 
-The servers tab lets you see the current status of the nodes making up your cluster.
+The **Services** view lets you see the current status of the nodes making up your cluster.
 You can group the nodes by type or by tier to get meaningful summary statistics. 
 
 ![servers](../assets/web-console-10-servers.png)
 
 ## Query
 
-The query view lets you issue [DruidSQL](../querying/sql.md) queries and display the results as a table.
-The view will attempt to infer your query and let you modify via contextual actions such as adding filters and changing the sort order when possible.
+The **Query** view lets you issue [Druid SQL](../querying/sql.md) queries and display the results as a table.
+The view will attempt to infer your query and let you modify the query via contextual actions such as adding filters and changing the sort order when possible.
+
+From the ellipsis menu beside **Run**, you can view your query history, see the native query translation for a given Druid SQL query, and set the [query context](../querying/query-context.md).
 
 ![query-sql](../assets/web-console-11-query-sql.png)
 
 The query view can also issue queries in Druid's [native query format](../querying/querying.md), which is JSON over HTTP.
-To send a native Druid query, you must start your query with `{` and format it as JSON.
+To send a native Druid query, start your query with `{` and format it as a JSON object.
```

Comment:
Don't all JSON objects begin with `{`?

Reply:
True. Rewording the previous sentence and removing the redundancy

Reply:
```suggestion
```